### ISSUE TITLE
`NumberInput`: `aria-describedby` should be dynamic

### ISSUE THREAD

**Issue by metonym** [carbon-design-system/carbon-components-svelte#2613]
React dynamically sets `aria-describedby` to point at the error, warning, or helper text element ID depending on state. The Svelte component always sets `aria-describedby` to a fixed `errorId` regardless of whether there's an error, warning, or helper text.
